pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedImportsRule.java

```diff
@@ -44,7 +44,7 @@
             .compile("@see\\s+(\\p{Alpha}?+\\w*)(?:#\\w*(?:\\(([\\w\\s,]*)\\))?)?");
```

Comment:
Wouldn't we run into the same problem with `@see SomeClass#method(MyArray[])`?
Could you please add a test for this as well?

Reply:
Sure, this totally makes sense.